kukur/source/piwebapi_da/piwebapi_da.py

```diff
@@ -256,6 +274,7 @@ def _get_data_url(self, session, selector: SeriesSelector) -> str:
             data_archive["Links"]["Points"],
             verify=self.__request_properties.verify_ssl,
             params=dict(
+                maxCount=str(self.__request_properties.max_returned_items_per_call),
```

Comment:
Why do we need `maxCount` here? We expect to receive only one series in the response.

Reply:
I ran into errors when testing with really low limits on the server (maxReturnedItemsPerCall = 5) and sending the `maxCount` ensured the call worked in these cases. Since this limit is not realistic, I agree it can be removed, but it also doesn't seem like a problem to keep it.